recipes/_install-linux.rb

```diff
@@ -39,8 +39,18 @@
     not_if 'apt-cache policy datadog-agent-base | grep "Installed: (none)"' if node['platform_family'] == 'debian'
   end
   # Install the regular package
-  package 'datadog-agent' do
-    version dd_agent_version
-    action node['datadog']['agent_package_action'] # default is :install
+  case node['platform_family']
+  when 'debian'
+    apt_package 'datadog-agent' do
+      version dd_agent_version
+      action node['datadog']['agent_package_action'] # default is :install
+      options '--force-yes' if node['datadog']['agent_allow_downgrade']
+    end
+  when 'rhel', 'fedora'
+    yum_package 'datadog-agent' do
+      version dd_agent_version
+      action node['datadog']['agent_package_action'] # default is :install
+      allow_downgrade node['datadog']['agent_allow_downgrade']
+    end
```

Comment:
Discussed this offline: since we're doing the same test in [`recipes/repository.rb`](https://github.com/DataDog/chef-datadog/blob/v2.5.0/recipes/repository.rb#L20-L21), the cookbook doesn't work on other platforms anyway, so no need to fall back on a default here.